samples/ck-rx65n/basic-sample/src/iotconnect_app.c

```diff
@@ -266,6 +295,15 @@ bool app_startup(NX_IP *ip_ptr, NX_PACKET_POOL *pool_ptr, NX_DNS *dns_ptr) {
 	auth_driver_context = config->auth.data.x509.auth_interface_context;
 #endif // ENABLE_RX_TSIP_AUTH_DRIVER_SAMPLE
 #endif  // IOTCONNECT_SYMETRIC_KEY
+#endif  // IOTCONNECT_INTERACTIVE_SYMMETRIC_KEY
+
+    printf("config->cpid: '%s'\r\n", config->cpid);
```

Comment:
This looks like debug. If you want to keep this printed, change "config->cpid:" to just "CPID:" and similar to the rest.

Reply:
Done